docs/pricing/am2-pricing.mdx

```diff
@@ -0,0 +1,262 @@
+
+- **Per-category:** Your on-demand budgets for categories such as errors, transactions, and replays, are set up individually so that any overages in one category won't consume the budget of another category.
+
+![On-Demand Budgets settings in Subscriptions](./img/on-demand-budgets.png)
+
+### Changing Plans Mid-cycle
+
+If you decide to increase your on-demand budget mid-month, Sentry will start processing additional data as soon as the increase becomes effective.
+
+If you want to decrease your on-demand budget mid-month, but have already consumed more data than the new desired amount allows, your budget will be lowered to match what you’ve already used and any new data will be rejected. Your on-demand budget will be decreased to the desired amount on your next billing date.
+
+If you upgrade from a Team to a Business plan mid-cycle, any existing on-demand budget will be recalculated based on Business tier pricing. This may mean that your on-demand budget will be consumed faster than it was before.
+
+In most cases, new budgets will take effect within minutes. Otherwise, we guarantee a maximum turnaround time of 24 hours.
+
+## Billing Periods
+
+Two different billing periods are available:
+
+- **Billing period** - Your subscription billing period, which is either monthly or annual. This is when we’ll charge the credit card on file for your base subscription and reserved volumes.
+- **On-demand period** - This reflects a one-month window in which we calculate on-demand charges. For monthly subscriptions, on-demand periods are aligned with your billing periods.
+
+Users with the role of [Billing or Owner](/organization/membership/) can find all previous invoices by going to the "Receipts" tab in **Settings > Subscription**.
+
+<Note>
+
+You must have either `Billing` or `Owner` permissions to access or make changes to information on the **Subscription** page. If you don't have those permissions, or you are not a member of the organization in Sentry, please reach out to a Billing member or an Owner to obtain invoices.
+
+</Note>
+
+## Plans & Free Trials
+
+Sentry has one free Developer plan and three paid plans: Team, Business, and Enterprise. You can find more information about each of these plans and what they include on our [pricing page](https://sentry.io/pricing/).
+
+Plan upgrades take effect immediately. Plan downgrades and cancellations are processed as the end of the current contract cycle and cannot be refunded. If you no longer want to be on a paid plan, you'll revert to the base Developer plan at the end of your current contract.
+
+All new Sentry accounts come with a 14-day free trial period. You'll have the option to upgrade and continue to use Sentry's latest features like Performance Monitoring at any time before your trial ends. Upgrades will be effective immediately. Users who haven't upgraded by the end of their free trial will remain on the free Developer plan with limited features.
+
```

Comment:
```suggestion
## Product Trials

In addition to the free Business trial available to new accounts, Sentry offers a one-time, 14-day product trial as a way to let your organization try out products that aren't part of your billing plan (such as Tracing and Session Replays). Product trials are specific to each product, so you can try one product at a time or multiple products at the same time.

Anyone who's on a Free, Team, or Business plan can initiate a product trial directly from the Sentry UI. Keep in mind that each org can only trial each product once.

![Image of UI with product trial pop up](./img/product-trial-pop-up-in-sentry-ui.png)

During your 14-day product trial, you'll be able to send us an unlimited number of events. If you continue to use the product after your trial period is up, any additional usage will be billed using your available reserved volume or your pay-as-you-go budget. You won't be billed for any data you've used during the 14-day trial period.

### Who's Eligible for a Product Trial and When?

**New accounts**

New accounts will be able to request a one-time product trial on day 15, after the initial 14-day Business trial has ended, or immediately after a plan has been upgraded to a paid plan.

**Free accounts**

Free accounts will be able to request a one-time product trial at any time.

**Existing Team or Business accounts**

- Existing Team or Business accounts will be able to request a one-time product trial for products that aren't already part of their billing plan at any time.

- Existing Team accounts that haven't opted in to a Business trial will have the option to request a Business trial and product trials for products that aren't already part of their billing plan at any time.
```
I think we should add this section since it applies to AM2 plans.